tests/install-infra.sh

```diff
@@ -81,7 +81,7 @@ function test_obi {
 	echo "wait 240s for data to fill"
 	sleep 240
 	for i in $(seq 1 6); do
-		[ $(kubectl get obi -n ${ARBITER_NS} | grep -v 'NAME' | awk '{print NR}' | tail -n1) -eq 8 ] && s=0 && break || s=$? && sleep 10
+		[ $(kubectl get obi -n ${ARBITER_NS} | grep -v 'NAME' | awk '{print NR}' | tail -n1) -eq 9 ] && s=0 && break || s=$? && sleep 10
```

Comment:
It also needs to be modified here.
https://github.com/kube-arbiter/arbiter/blob/main/tests/e2e/observer_test.go#L35

Reply:
Add more here: https://github.com/kube-arbiter/arbiter/pull/36/commits/454b061a8174d3abd434e3d1303724f2c1f8e904
We should think about how to make it easy to add more OBI test samples and don't break the tests.